docs/content/add-plugins/stripe.md

```diff
@@ -1,3 +1,109 @@
+
+```bash
+# yarn
+yarn add medusa-payment-stripe
+
+
+# npm
+npm install medusa-payment-stripe
+```
+
+Then in your `medusa-config.js` , add the plugin to your `plugins` array:
+
+```javascript
+module.exports = {
+ // ... other options
+   plugins: [
+       // ... other plugins
+       {
+            resolve: `medusa-payment-stripe`,
+            options: {
+                api_key: STRIPE_API_KEY,
+                webhook_secret: STRIPE_WEBHOOK_SECRET,
+            },
+       },
+   ];
+}
+```
+
+Now head over to [Stripe](https://stripe.com/) and create your account. You can then click API Keys on your dashboard, and here you will see two keys. We suggest using the test environment during development, and therefore you should make sure that you are seeing the keys to the test environment (These keys start with `pk_test_` and `sk_test_` respectively). 
+
+Now open your `.env` file for the Medusa project and store your *secret key* in the `STRIPE_API_KEY` variable:
+
+```bash
+# .env
+STRIPE_API_KEY=<your key here>
+```
+
+
+> Note: For production you should also create a webhook on Stripe (also available on your dashboard) and store its secret in the `STRIPE_WEBHOOK_SECRET` variable. We will go into detail with this in a later guide.
+
+Then navigate to your Gatsby starter project and open the `.env.development` file and store your *publishable key* in the `GATSBY_STRIPE_KEY` variable:
+
+```bash
+# .env
+GATSBY_STRIPE_KEY=<your key here>
+```
+
+### What’s next?
+
+At this point we have set everything up, and the Stripe payment provider is now enabled in your Medusa project. So, go ahead and start up your medusa project, the gatsby starter, and the admin panel!
+
+However, as Medusa allows for different payment providers in different regions (and multiple providers in each) we should first make Stripe a valid payment option in our default region. To do so, open the admin panel (`http://localhost:7000`) login, and navigate to `Settings > Region settings > Edit Default Region`. Here you should now be able to select stripe as a payment provider:
+
+<center>
+
+![Change payment provider](https://i.imgur.com/mVIDYz4.png)
+</center>
+
+After doing this, and clicking save, we are ready to accept payments using Stripe. So, navigate to your storefront (`http://localhost:8000`) and go through the checkout process:
+
+<center>
+
+![Checkout process](https://i.imgur.com/2pm9JyR.gif)
+</center>
+After doing so, you should be able to see an uncaptured payment in the Stripe. Here, you navigate to the payments tab, where you should see the following (depending on your choices during the checkout process):
+
+<center>
+
+![Uncaptured payment](https://i.imgur.com/LX6UR40.png)
+</center>
+
+To then capture the payment, navigate back to the admin panel (`http://localhost:7000/`), and dig into the relevant order, and capture the payment:
+
+<center>
+
+![Capture payment](https://i.imgur.com/y5UxxpS.gif)
```

Comment:
I really like the use of a GIF! This is a personal preference, but it might be an idea to speed it up a bit. Skip selecting a product, maybe pre-save an address, so it gets to the point a bit faster :)